src/Extension/FluentExtension.php

```diff
@@ -835,4 +846,118 @@ public function cacheKeyComponent()
+//                        $titleClasses .= ' fluent-modified-value';
+//                        $modifiedTitle = 'Modified from default locale value - click to reset';
+//                    }
+
+                    $field->setTitle(
+                        DBHTMLText::create()->setValue(
+                        sprintf(
+                            '<span class="%s" title="%s">%s</span>%s',
+                            $titleClasses,
+                            $modifiedTitle,
+                            $language,
+                            $title
+                        ))
+                    );
+
+                    // Set the default value to the element so we can compare it with JavaScript
+//                    if (Locale::getDefault() !== $locale) {
+//                        $field->setAttribute(
+//                            'data-default-locale-value',
+//                            $this->owner->{Fluent::db_field_for_locale($field->getName(), Locale::getDefault())}
+//                        );
+//                    }
+
+                    $field->addExtraClass('LocalisedField');
+                }
+
+                // Remove translation DBField from automatic scaffolded fields
+//                foreach (Fluent::locales() as $locale) {
+//                    $fieldName = Fluent::db_field_for_locale($translatedField, $locale);
+//                    $fields->removeByName($fieldName, true);
+//                }
+            }
+        }
+
+        $this->addLocaleIndicatorMessage($fields);
+    }
+
+    /**
+     * Adds a UI message to indicate whether you're editing in the default locale or not
+     *
+     * @param  FieldList $fields
+     * @return $this
+     */
+    protected function addLocaleIndicatorMessage(FieldList $fields)
+    {
+        if (Config::inst()->get(__CLASS__, 'disable_current_locale_message')) {
+            return $this;
+        }
+
+        // If the field is already present, don't add it a second time
+        if ($fields->fieldByName('CurrentLocaleMessage')) {
+            return $this;
+        }
+
+        $localeNames     = Locale::get()->map('Locale', 'Title')->toArray();
+        $defaultLocale   = Locale::getDefault();
+        $isDefaultLocale = ($defaultLocale === Locale::getCurrentLocale());
+        $defaultLocaleKey = $defaultLocale ? $defaultLocale->Locale : '';
+        $currentLocale = Locale::getCurrentLocale();
+        $currentLocaleKey = $currentLocale ? $currentLocale->Locale : '';
+        $messageClass    = ($isDefaultLocale) ? 'good' : 'notice';
+        $message         = ($isDefaultLocale)
+            ? _t('Fluent.DefaultLocale', 'This is the default locale')
+            : _t('Fluent.DefaultLocaleIs', 'The default locale is') . ' ' . $localeNames[$defaultLocaleKey];
+
+        $fields->unshift(
+            LiteralField::create(
+                'CurrentLocaleMessage',
+                sprintf(
+                    '<p class="message %s">' . _t(
```

Comment:
Remove `message` and let it be pushed in by `$messageClass` (see previous comment)